### PR TITLE
Structure for tests

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Hi, this PR starts adding some structure for tests as we have them in Scikit-HEP packages.

Starts addressing https://github.com/scikit-hep/histoprint/issues/12 and https://github.com/scikit-hep/histoprint/issues/4 to some extent.

With this I can see the tests running issuing `python -m pytest tests/test.py -vv` under the main directory. Of course there is no CI set up here, so that will have to come ...

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Hi @ast0815, is this easy PR being forgotten …?

**Comment by eduardo-rodrigues** [comment]
To be honest, @ast0815 I would prefer to do that or have it done in a different PR. For one, that's a different matter from this PR on structure. Also, I'm totally overloaded at the moment and do not exactly know how to do what you suggest, so no way I can do it "just now". But if you create an issue for your suggestion I will try and help next week.

If agreed, just approve, squash and merge :-).

File: .github/workflows/pythontests.yml [hunk 1]
```diff
@@ -27,20 +27,20 @@ jobs:
...
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-        pip install -e .
+        python -m pip install -e .[test]
     - name: Install optional dependencies for tests
       run: |
         pip install boost-histogram uproot pandas || true
     - name: Run tests
       run: |
-        python -m histoprint.test
+        python -m pytest tests/test.py
```
Comment (ast0815) [inline .github/workflows/pythontests.yml:36]:
```suggestion
        python -m pytest -s tests/test.py
```
This should do it.

Comment (eduardo-rodrigues) [reply .github/workflows/pythontests.yml:36]: Oh, then I think I misunderstood the scope of your comment. Sure, that prints it all in the logs of the CI.

**Comment by ast0815** [comment]
Perfect, thanks!

**Pull request merged**